src/util/thpool/thpool.c

```diff
@@ -406,6 +417,10 @@ static int jobqueue_init(jobqueue *jobqueue_p) {
 	pthread_mutex_init(&(jobqueue_p->rwmutex), NULL);
 	bsem_init(jobqueue_p->has_jobs, 0);
 
+	if(!Config_Option_get(Config_MAX_QUEUED_QUERIES, &jobqueue_p->max_queued_queries)) {
+		return -1;
+	}
+
```

Comment:
Replace this with `UINT64_MAX` to indicate unlimited queue size
`jobqueue_p->cap = UINT64_MAX`;